src/Map.jsx

```diff
@@ -38,17 +35,87 @@ class Map extends Component {
         coordinates: { lon: 0, lat: 0 },
         info: { vid: '', heading: 0 },
       },
+      geojson: muniRoutesGeoJson,
     };
   }
 
+  componentWillMount() {
```

Comment:
If possible, let's use componentDidMount as then it'll only run once - and it's better practice to initialize things (lines 43 and 45) in the constructor. See https://github.com/facebook/react/issues/7671